### ISSUE TITLE
Option for AuthMe MC Integration to must use same email for site registering as in-game

### ISSUE THREAD

**Issue by TheJoshue** [NamelessMC/Nameless#2589]
### Describe the feature

Adding an option right above/below this option on the image (this is at `StaffCP` > `Integrations` > `Minecraft` > `AuthMe Integration`)
![image](https://user-images.githubusercontent.com/56804505/151726724-258b81ef-79d1-4d2c-94cf-81bd1d80ece4.png)

An option "`Require users to use same email from in-game`"

If this option is enabled, when users go to NamelessMC site and input their in-game username & password, it'll always ask them to input email. However people can input any email (that isn't used) which is making it difficult for people to remember their account email because they'll have different in-game and on site.

This is why this option (or even making AuthMe integration function like this by default) is very useful and should be considered.

Basically making email address required to be inputted the same as in-game (just like username password, you must input those exactly from in-game).

If player don't have email set to his account or he input other email while registering, the message will be shown on register page that he either didn't add email in-game or the inputted email isn't correct to his account.

**Comment by Aberdeener** [comment]
This is probably out of scope of our AuthMe integration

**Comment by TheJoshue** [comment]
> This is probably out of scope of our AuthMe integration, it is not something we would like to spend much time on

ok then remove requirement of adding email address upon registering an account with AuthMe Integration enabled
because you can't reset password via "Forgot Password" (which will send you email where you can *reset* your password but it won't affect anything because your account is tied to AuthMe account user/pass and can only be changed in-game) or via "Change Password" in Profile Settings (also this needs to be removed if authme integration is enabled, or replaced with custom text)

**Comment by TheJoshue** [comment]
> > This is probably out of scope of our AuthMe integration, it is not something we would like to spend much time on
> 
> ok then remove requirement of adding email address upon registering an account with AuthMe Integration enabled because you can't reset password via "Forgot Password" (which will send you email where you can _reset_ your password but it won't affect anything because your account is tied to AuthMe account user/pass and can only be changed in-game) or via "Change Password" in Profile Settings (also this needs to be removed if authme integration is enabled, or replaced with custom text)

any thoughts on this?

**Comment by Aberdeener** [comment]
We will get to this when we get to this

**Comment by samerton** [comment]
The reason email address is mandatory after signing up with AuthMe is because (at least when the integration was developed) emails are not mandatory within AuthMe itself.

An email address is a required field in Nameless so we must always capture it - we can't guarantee AuthMe has one set.

**Comment by TheJoshue** [comment]
> 



> The reason email address is mandatory after signing up with AuthMe is because (at least when the integration was developed) emails are not mandatory within AuthMe itself.
> 
> An email address is a required field in Nameless so we must always capture it - we can't guarantee AuthMe has one set.

ok i understand, but what about instead of doing that, if AuthMe Integration is enabled -> don't ask for email at all (but tell player if they want to add email they will need to do it in-game)

because even if you have email address to an account it's useless -> you can't use it to receive email to reset forgotten password (you CAN, you will receive email, but it won't affect your password at all because it's forced to be looked from authme database for your account)
